data/migrations/20220117123847_case_mgmt_schema.js

```diff
@@ -1,7 +1,7 @@
 exports.up = function (knex) {
   return knex.schema
     .createTable('locations', (t) => {
-      t.increments('ID').notNullable().unique().primary();
+      t.increments().notNullable().unique().primary();
```

Comment:
I'm a bit confused by this change. Shouldn't the ID objects have names?

Reply:
'ID' causes an error in knex when it tries to read the schema. And an  empty`.increments()` populates as an 'id'  in the schema when the code is being run

Reply:
Would a more descriptive name like 'location_id" work?

Reply:
No the model is looking for the auto increment  of 'id'.

Reply:
Makes sense. Are the id's eventually supposed to be UUID's?